src/Cli.Tests/ModuleInitializer.cs

```diff
@@ -79,6 +79,8 @@ public static void Init()
         VerifierSettings.IgnoreMember<RuntimeConfig>(options => options.EnableAggregation);
         // Ignore the AllowedRolesForHealth as that's unimportant from a test standpoint.
```

Comment:
Please mention which commits were cherry picked in the PR description. 

Also, describe if you had to resolve any merge conflicts when you cherry picked them.
Finally, please create a separate PR for cherry picking Hotchocolate migration.